yellowbrick/regressor/residuals.py

```diff
@@ -285,7 +279,20 @@ def finalize(self, **kwargs):
         self.ax.legend(loc="best", frameon=True)
 
 
-def prediction_error(model, X, y=None, ax=None, alpha=0.75, is_fitted="auto", **kwargs):
+def prediction_error(
+        model,
+        X_train,
+        y_train=None,
```

Comment:
I believe that `y_train` is required, even though the `show` method in PredictionError has `y=None` this is probably an error, I can't think of a reason why `y_train` should be None -- can you?